src/App.tsx

```diff
@@ -10,6 +10,7 @@ import Quiz from './ModuleList/Quiz';
 import Lesson from './ModuleList/Lesson';
 import { useFeatureFlags } from './FeatureFlags';
 import Layout from './layout';
+import ErrorBoundary from './common/components/ErrorBoundary/ErrorBoundary';
```

Comment:
Can you add an `index.ts` in the `src/common/components/ErrorBoundary/` that exports the `ErrorBoundary` components. This way out import would look like 
```suggestion
import ErrorBoundary from './common/components/ErrorBoundary';
```